docker/erlang.config

```diff
@@ -0,0 +1,76 @@
+%% -*- mode: erlang -*-
+[
+
+ {exometer, [{predefined, [
+    {[erlang, memory], {function, erlang, memory, [], value, []}, []},
+    {[erlang, system_info], {function, erlang, system_info, ['$dp'], value, [process_count]}, []},
+    {[erlang, statistics], {function, erlang, statistics, ['$dp'], value, [run_queue]}, []},
+    {[erlang, io], {function, erlang, statistics, [io], match, {{'_', input}, {'_', output}}}, []}
+   ]}
+ ]},
+
+ {mnesia, [
+    {dir, "/tmp/mnesia"}
+ ]},
```

Comment:
Some are removed (console log), others are commented out (access log). I suppose since both those lines could be removed instead of commented, that a sed /foo/d would work for them. 

Truthfully though I'm not a huge fan of sed'ing config files like these and do it for zotonic.conf only because values need to be inserted. That said I'm also not a fan of shipping config files so I guess I'll sed the file into compliance.